sphinx/locale/ar/LC_MESSAGES/sphinx.js

```diff
@@ -36,7 +36,14 @@ Documentation.addTranslations({
         "Search": "\u0628\u062d\u062b",
         "Search Page": "\u0635\u0641\u062d\u0629 \u0627\u0644\u0628\u062d\u062b",
         "Search Results": "\u0646\u062a\u0627\u0626\u062c \u0627\u0644\u0628\u062d\u062b",
-        "Search finished, found ${resultCount} page(s) matching the search query.": "",
+        "Search finished, found one page matching the search query.": [
+            "",
+            "",
+            "",
+            "",
+            "",
+            ""
+        ],
```

Comment:
@AA-Turner @hugovk hrm.  there's something odd going on in these automated translation updates (re: search result plurality, perhaps?)

Reply:
Ah!  Maybe the number of empty-strings represents the number of plural forms expected in each language.

Reply:
...yep, looks like it.  Apologies: probably a false alarm by me.

That said, I'll try to test one of these multiple-empty-string internationalizations to confirm that it works (falls-back to the untranslated original) as expected.

Reply:
Non-empty translations (`fr`):

![image](https://github.com/user-attachments/assets/c565255e-85b8-4be6-aa14-d9fe86dca4c5)

Empty translations (`ro`):

![image](https://github.com/user-attachments/assets/712b8a20-fb70-4761-89e3-83d63e80fb93)

In short: when these translation entries are empty, no result-count message (not even a fallback-language one) is displayed to the user.

I'm not yet sure whether this is a regression, nor whether it's worth considering release-critical.

Reply:
Ok - the behaviour was the same in v7.4.7:

![image](https://github.com/user-attachments/assets/05953812-2349-40ca-949d-5458b4a29887)

So; not a regression, and not release-critical.  Only a small oddity that we might want to address (or perhaps it is genuinely better not to display any result-count output in this area in some localized versions, for some reason).

Reply:
@n-peugnet I think this might be a quirk related to differenes in fallback behaviour between rendering of static HTML (where we _do_ allow untranslated text, such as "Searching for multiple words only shows matches that contain all words"), as compared to the fallback behaviour determined in the JavaScript code that runs on a few pages (such as the search page -- where an empty translation appears not to display a fallback untranslated text at all, at least in this example).